JustFakeIt.Tests/FakeServerScenarios.cs

```diff
@@ -37,26 +34,40 @@ public void FakeServer_ExpectGetReturnsString_ResponseMatchesExpectation()
         {
             const string expectedResult = "Some String Data";
 
-            var port = Ports.GetFreeTcpPort();
-
-            var baseAddress = "http://localhost:" + port;
-
             const string url = "/some-url";
 
-            using (var fakeServer = new FakeServer(port))
+            using (var fakeServer = new FakeServer())
             {
                 fakeServer.Expect.Get(url).Returns(expectedResult, new WebHeaderCollection { { "foo", "bar" } });
                 fakeServer.Start();
 
-                var client = new HttpClient {BaseAddress = new Uri(baseAddress)};
-                var result = await client.GetAsync(url);
+                var result = await fakeServer.Client.GetAsync(url);
 
                 result.Content.ReadAsStringAsync().Result.Should().Be(expectedResult);
                 result.Headers.Should().ContainSingle(x => x.Key == "foo");
             }
         }
 
         [Fact]
+        public async Task FakeServer_ExpectGetWithResponseHeadersSpecifiedFluently_ResponseMatchesExpectionAndHasHeaders()
+        {
+            const string expectedResult = "Some String Data";
+
+            const string url = "/some-url";

```

Comment:
I know it's nick-picking, and the mistake is being propagated everywhere, but this is badly named; it's not a URL - it's a path